test/unit/sdk.test.js

```diff
@@ -747,27 +728,16 @@ describe('sdk', () => {
     };
 
     // expected OAuth link
-    const expectedLink =
```

Comment:
why did we removed this?

Reply:
Unused due to other changes

Reply:
It was unused, but I have re-added it. See other comment.